### PR TITLE
[HIGH PRIORITY] Fix CreateSchema method

### PR DESCRIPTION
**Pull request by prdlk**
There were a couple of things getting blocked in the Schema module on chain.

## Changes

* Fixed creator string being set to `did:snr:id` value instead of Wallet public key

## Checklist

* [x] Unit tests
* [x] Documentation

## Notes

I was going to definitely do a lot more changes with the API, refactoring the Query structure, and the actual methods being put on chain - however we need the one key fix that will help us see everything come together

Fixes
Connects

 Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here.

### CODE REVIEW

**Review by mcjcloud — APPROVED** [review]
lgtm

File: x/schema/keeper/msg_server_schema.go [hunk 1]
```diff
@@ -14,33 +14,6 @@ import (
 
 func (k msgServer) CreateSchema(goCtx context.Context, msg *types.MsgCreateSchema) (*types.MsgCreateSchemaResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	err := msg.ValidateBasic()
```
Comment (ntindle) [inline x/schema/keeper/msg_server_schema.go:17]: why toss this one?

File: x/schema/keeper/msg_server_schema.go [hunk 2]
```diff
@@ -14,33 +14,6 @@ import (
...
-
-	k.Logger(ctx).Info("msg validation successful")
-
-	accts := msg.GetSigners()
-	if len(accts) < 1 {
-		return nil, sdkerrors.ErrNotFound
-	}
-
-	creator_did := msg.GetCreatorDid()
-	k.Logger(ctx).Info(fmt.Sprintf("Creating schema for creator did %s", creator_did))
```
Comment (ntindle) [inline x/schema/keeper/msg_server_schema.go:30]: Keep thsese ones

File: x/schema/keeper/msg_server_schema.go [hunk 3]
```diff
@@ -14,33 +14,6 @@ import (
...
-	}
-
-	if err != nil {
-		return nil, sdkerrors.Wrapf(err, "Error while pinning schema definition to storage")
-	}
-
-	if err != nil {
-		return nil, sdkerrors.Wrapf(err, "Error while persisting schema fields")
-	}
-
```
Comment (ntindle) [inline x/schema/keeper/msg_server_schema.go:43]: :???: how will these be hit ever?

**Review by ntindle — APPROVED** [review]

**Pull request merged**